specification/quota/resource-manager/Microsoft.Quota/stable/2021-03-15/quota.json

```diff
@@ -0,0 +1,1296 @@
+          "description": "URL to get the next page of items."
+        }
+      }
+    },
+    "OperationResponse": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "display": {
+          "$ref": "#/definitions/OperationDisplay"
+        },
+        "origin": {
+          "type": "string"
+        }
+      }
+    },
+    "OperationDisplay": {
+      "type": "object",
+      "properties": {
+        "provider": {
+          "type": "string",
+          "description": "Provider name."
+        },
+        "resource": {
+          "type": "string",
+          "description": "Resource name."
+        },
+        "operation": {
+          "type": "string",
+          "description": "Operation name."
+        },
+        "description": {
+          "type": "string",
+          "description": "Operation description."
+        }
+      }
+    },
+    "CommonResourceProperties": {
+      "type": "object",
+      "description": "Resource properties.",
+      "properties": {
+        "id": {
+          "description": "Resource ID.",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "Resource name.",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Resource type. \"Microsoft.Quota/quotaLimits\""
+        }
+      }
+    },
+    "CurrentQuotaLimit": {
+      "type": "object",
+      "description": "Quota limit.",
+      "x-ms-azure-resource": true,
+      "properties": {
+        "quotaInformation": {
+          "description": "Quota information details.",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/CurrentQuotaLimitBase"
+        },
+        "properties": {
+          "description": "Additional properties for the quota request status for the specified resource.",
+          "$ref": "#/definitions/QuotaRequestStatusDetails",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "CurrentQuotaLimitBase": {
+      "type": "object",
+      "description": "Quota limit.",
+      "x-ms-azure-resource": true,
+      "properties": {
+        "id": {
+          "description": "The resource Id.",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "description": "The resource type.",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "The resource name.",
+          "type": "string",
+          "readOnly": true
+        },
+        "properties": {
+          "description": "Quota properties for the specified resource.",
+          "$ref": "#/definitions/QuotaProperties"
+        }
+      }
+    },
+    "QuotaProperties": {
+      "description": "Quota properties for the specified resource.",
+      "type": "object",
+      "properties": {
```

Comment:
>properties [](http://example.com/codeflow?start=7&length=10)

We need "provisioningState" here since PUT returns 201 for async operation.

If you don't want to return provisionState, please have PUT return 202 instead.

Reply:
I will change it 202.